connect-location/build.gradle

```diff
@@ -30,7 +30,9 @@ android {
 dependencies {
     api project(':connect-api')
```

Comment:
Forgot to ask this in the previous PR: do we really need to expose `connect-api` via `connect-location`? If the developers chose to use connect-location without ConnectButton, do we consider that a valid use case?

Reply:
no, that's a good point.